src/main/webapp/resources/js/components/sequence-files/SingleEndFileRenderer.jsx

```diff
@@ -60,23 +67,28 @@ export function SingleEndFileRenderer({
                           )
                     }
                     target="_blank"
+                    className="t-file-label"
                   >
                     {file.fileInfo.label}
                   </a>
                 ) : (
-                  <span>{file.fileInfo.label}</span>
+                  <span className="t-file-label">{file.fileInfo.label}</span>
                 )}
 
                 <Space direction="horizontal" size="small">
                   {file.fileType === "assembly"
                     ? null
                     : getProcessingState(file.fileInfo.processingState)}
-                  <span style={{ marginRight: SPACE_XS }}>
+                  <span
+                    style={{ marginRight: SPACE_XS }}
```

Comment:
Why adding more space inside a space?

Reply:
Turns out it's something I had added when I was adding the Remove button onto each file line rather than for each sequencing object. I willl remove the space

Reply:
Removed in a0030449ca44d14e86faa4e3ece5b118bf115cc1